### ISSUE TITLE
Reading tables

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4853]
**Reported by supanut2000 on 2015-01-27 13:26**
Hi,
When using Microsoft Word 2010, whenever I save a file, close Word and reopen the file, NVDA no longer reads the contents of a table, but insteads plays a sound, indicating that it can't read the table. This only happens if I save the file and then close Microsoft Word and open the file again; unsaved documents appears to work fine.

**Comment by Adriani90** [comment]
@supanut2000 can you still reproduce this with NVDA 2018.4 RC1? If yes, please provide an MS Word document for reproducing this issue.

**Comment by Adriani90** [comment]
I guess this could also occur if the document is protected. When you open the document, hit f6 and then tab to enable editing. For me I cannot reproduce this issue with NVDA 2019.1.1, word 2010 and Word 2016 / 365. I have tested with protected and unprotected documents.
If you are still having this problem plese comment on it and we can reopen the discussion.
